app/Http/Controllers/GeneralLedgerAccountsController.php

```diff
@@ -34,52 +36,87 @@ public function create()
      * @return \Illuminate\Http\Response
      */
     public function store(Request $request)
-    {
-        //
-    }
+    {}
 
     /**
      * Display the specified resource.
      *
-     * @param  \App\GeneralLedgerAccounts  $generalLedgerAccounts
+     * @param  \App\GeneralLedgerAccounts  $chart_of_account
      * @return \Illuminate\Http\Response
      */
-    public function show(GeneralLedgerAccounts $generalLedgerAccounts)
-    {
-        //
-    }
+    public function show(GeneralLedgerAccounts $chart_of_account)
+    {}
 
     /**
      * Show the form for editing the specified resource.
      *
-     * @param  \App\GeneralLedgerAccounts  $generalLedgerAccounts
+     * @param  \App\GeneralLedgerAccounts  $chart_of_account
```

Comment:
help text is freaking misleading :laughing:

Reply:
will rename